vmdb/app/models/host.rb

```diff
@@ -1244,15 +1244,21 @@ def detect_discovered_os(ost)
   def detect_discovered_hypervisor(ost, ipaddr)
     log_header = 'MIQ(Host.detect_discovered_hypervisor)'
     find_method = :find_by_ipaddress
+
+    # TODO: break this apart, we probably want to only do the namelookup for hyperv, esx
+    hostname = Socket.getaddrinfo(ipaddr, nil)[0][2]
```

Comment:
By the way, `Resolv` is the worst shortcut ever.  :trollface:  
It ranks up there with shortening external but not management or system (ExtManagementSystem)  :laughing: